crates/wasmparser/src/validator/operators.rs

```diff
@@ -2291,10 +2333,12 @@ where
         // FIXME(#924) this should not be conditional based on enabled
         // proposals.
         if self.features.function_references {
```

Comment:
FWIW I think you may have fixed this and this branch may be able to be deleted

(awhile back, not in this PR specifically)

Reply:
I honestly don't know what this comment is about because that issue is fixed but also had nothing to do with this comment afaict?

Reply:
Ah no worries, I'll remove it later. I got confused and thought this comment was here for typechecks possibly becoming exponential. I was wrong though and this was here for another reason, unrelated to this PR entirely.